sound/soc/sof/pm.c

```diff
@@ -411,6 +411,26 @@ int snd_sof_runtime_resume(struct device *dev)
 }
 EXPORT_SYMBOL(snd_sof_runtime_resume);
 
+int snd_sof_set_d0_substate(struct snd_sof_dev *sdev,
+			    enum sof_d0_substate d0_substate)
```

Comment:
Indent fix needed here as well.

Reply:
I think this is already aligned?  they are both start from the 29th character.

Reply:
Not in the web view here in Github but l trust your editor more.

Reply:
@singalsu yes, I can trust the web view, so it's great that you used it to find an issue for me.